.github/workflows/publish-dist.yml

```diff
@@ -29,9 +28,11 @@ jobs:
       - name: Commit to dist repo
         run: |
           set -ex
+          mkdir dist-repo
           cd dist-repo
+          tar --strip-components=1 -xf ../cockpit-*.tar.xz
```

Comment:
Interestingly, it would be possible for the builder to have sent us more than one tarball, in which case this would expand to the name of the first one as a parameter to `-f` and the name of the following ones as parameters understood to be the list of files to extract.

I don't *think* there's a potential attack there, but it's certainly a bit unexpected.

Reply:
I added a validation that there is exactly one tarball.